src/components/Pivot/Pivot.tsx

```diff
@@ -0,0 +1,74 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import { Pivot as FabricPivot, PivotItem as FabricPivotItem } from 'office-ui-fabric-react/lib/Pivot';
+
+import './Pivot.css';
+
+export interface PivotItem {
+  ariaLabel?: string;
+  key: string;
+  text: string;
+}
+
+export interface PivotProps extends NestableBaseComponentProps {
```

Comment:
we're not rendering children, so this should extend `BaseComponentProps` instead